DragonSlayer.cpp

```diff
@@ -5,16 +5,32 @@
 
 //DragonSlayer::getName
 
+const std::string& DragonSlayer::getName() {
+    return name;
+}
+
+
+std::string DragonSlayer::getStats() 
+{
+    //return Character::getStats();
+}
+
+
 void DragonSlayer::attack(Character& other)
 {
     std::cout << name << " is attacking " << other.getName() << " !!" << std::endl;
     if( auto* dragon = dynamic_cast<Dragon*>(&other) )
     {
-        assert(false);
+        //assert(false);
         //DragonSlayers get a 10x boost when attacking dragons, from their attack item.
         //so they should USE their attack item before attacking the dragon... 
         //note that items are single-use only, so you need to reset it after use.  
         //look in the Character class for how the other item types are reset after use.
+
+        
+
+        getHelpfulItems();
+
```

Comment:
This is not the solution.
Look at how items are used in `Character.cpp` for hints regarding what to write here.
- [ ] revise